enigma-swing/src/main/java/cuchaz/enigma/gui/elements/MenuBar.java

```diff
@@ -447,6 +452,8 @@ public void onStartServerClicked() {
 			JOptionPane.showMessageDialog(this.gui.getFrame(), e.toString(), I18n.translate("menu.collab.server.start.error"), JOptionPane.ERROR_MESSAGE);
 			this.gui.getController().disconnectIfConnected(null);
 		}
+
+		throw new RuntimeException();
```

Comment:
What does this do?

Reply:
oh oops I was using that to test
will fix once I get home